core-services/release-controller/_repos/ocp-4.10-rhel8.repo

```diff
@@ -1,3 +1,15 @@
+[rhel-8.4-baseos]
```

Comment:
Can you add the repos below the existing ones so that we don't change the liveness probe in the file above?

Reply:
Done